api/src/main/java/org/openmrs/module/appointmentschedulingui/reporting/library/AppointmentSchedulingUIDataSetDefinitionLibrary.java

```diff
@@ -102,6 +105,11 @@ public AppointmentDataSetDefinition getDailyAppointmentsDataSetDefinition() {
         dsd.addColumn("startDatetime", new AppointmentStartDateDataDefinition(), "", null);
         dsd.addColumn("startTimeFormatted", new AppointmentStartDateDataDefinition(), "", new DateConverter(AppointmentSchedulingUIConstants.TIME_FORMAT));
         dsd.addColumn("endTimeFormatted", new AppointmentEndDateDataDefinition(), "", new DateConverter(AppointmentSchedulingUIConstants.TIME_FORMAT));
+        dsd.addColumn("creator", new AppointmentCreatorDataDefinition(), "",
+                new ChainedConverter(
+                        new PropertyConverter(String.class, "person"),
+                        new PropertyConverter(Person.class, "personName"),
+                        new ObjectFormatter("{givenName} {familyName}")));
```

Comment:
Instead of these 2, I would just use "new ObjectFormatter()", which by default if given a Person will return Person.getFullName(), which is likely what you want.